proxies/shared/partials/Partial.Target.FaultRules.xml

```diff
@@ -19,9 +19,9 @@
         </Condition>
     </Step>
     <Step>
-        <Name>RaiseFault.GenericErrorDetails</Name>
```

Comment:
`RaiseFault.GenericErrorDetails` wasn't really required, the response is just built in JS.